### DISCUSSION TITLE
Welcome 👋🏼

### DISCUSSION THREAD

**Discussion by PatrickHeneise** [cyprusjs/CyprusJS#34] [Announcements]
Welcome everyone to CyprusJS!

If you're looking for fellow JavaScript Enthusiasts in your area, you can open a conversation in https://github.com/cyprusjs/CyprusJS/discussions/categories/general

**Proposals** If you have something interesting you want to talk about, please open open a discussion in https://github.com/cyprusjs/CyprusJS/discussions/categories/proposals

**Ideas** If you have an idea of how to make CyprusJS better, please start a conversation in https://github.com/cyprusjs/CyprusJS/discussions/categories/ideas

**Questions** Any questions you can put in our Q&A channel https://github.com/cyprusjs/CyprusJS/discussions/categories/q-a

**Comment by akarelas** [comment]
Hello. I'm Alex, Perl + JavaScript programmer living in Limassol, and wish to be added to the organization.

**Reply by PatrickHeneise** [reply]
I'm very happy [gitevents](https://gitevents.org) works so well and invited you automatically :) Welcome!
